Review ch5 (#544)
worldbank/dime-data-handbook: chapters/5-processing.tex

 may be ambiguous in many raw datasets.
This section will present what we call a \textit{tidy} data format,
which is, in our experience, the ideal format to handle tabular data.

> Although data quality assurance and data tidying are often done at the same time,
> particularly one it comes to guaranteeing that observations are uniquely identified,
> we will treat it as the first step in data cleaning.
> This is because quality assurance can only be finalized using tidied data.
> In practice, both tidying and quality monitoring should be done continuously as data is received.
> 

Reviewer: ```suggestion
We will treat tiding data as the first step in data cleaning,
even though in practice, 
both tidying and quality monitoring should be done simultaneously as data is received.
This is because quality assurance can only be finalized using tidied data,
when it is guaranteed that each observations are uniquely identified.
```
I am not trying to change what you are saying, but I had to read your first sentence a few times before I got what you were saying. I think this gets to the point you are trying to make quicker.

Data quality assurance requires a combination of real-time data checks
and back-checks or validation audits, which often means tracking down
the people whose information is included in the dataset.
Whether your team is conducting a survey or 

> you are receiving data from a partner,

Reviewer: ```suggestion
you are receiving administrative data from a partner,
```
To make it more clear that is what we mean, otherwise it sounds as we just as well could mean survey data from a partner

a social network module, drop all names.
If confidential information is strictly required for the analysis and cannot be
masked or encoded,

> additional precautions need to be taken
> unless everyone accessing it is always working in a secure environment,
> which is rarely the case.
> In most cases,
> it will be necessary to keep at least a subset of the data 
> stored in an encrypted format through the data analysis process.
> This data will need to be decrypted and loaded to a \textit{secure environment}
> every time you run code to analyzed or process it.

Reviewer: ```suggestion
then at least the confidential part of the data is required 
to remain encrypted 
and only be decrypted when used during in the data analysis process.
It is never an excuse to store confidential data unencrypted
or saved in any other insecure way,
just because it is needed in the analysis process.
```
I think we are confusing things by bringing in _secure environment_. What we really are saying is that it is not an excuse to save confidential data insecurely just because your analysis scripts reads those files. Other parts of the book covers the details of whats secure or not. Does this sound good? I think this would have prevented Jim's confusion.

Author: ```suggestion
at least the confidential part of the data is required 
to remain encrypted 
and only be decrypted when used during in the data analysis process.
Using confidential data in the analysis process
does *not* justify storing or sharing it in an insecure way.
```

Reviewer: Does `*not*` formatting work in LaTeX? or is this markdown formatting baked in?

Reviewer: I think the "then" on line 613 should still be there, but a professional reviewer is the person best suited to decide what is best. Was it intentional that you removed it? Apart from that I think your last version is ready to be committed.

Author: I added it back. I think it's optional, and I removed it just for stylistic reasons, but I don't feel strongly about it
